misc/bazel/internal/install.py

```diff
@@ -42,8 +44,20 @@
 script = runfiles.Rlocation(opts.pkg_install_script)
 
 if destdir.exists() and opts.cleanup:
-    shutil.rmtree(destdir)
-
+    if platform.system() == 'Windows':
+        # On Windows we might have virus scanner still looking at the path so
+        # attempt removal a couple of times sleeping between each attempt.
+        for attempt in [1, 2]:
+            try:
+                shutil.rmtree(destdir)
+                break
+            except OSError as e:
+                if e.winerror == 32:
```

Comment:
can we give a name for this 32 magic number?

Reply:
I don't think python exposes a public api for this, they themselves do this: https://github.com/python/cpython/blob/8ad6067bd4556afddc86004f8e350aa672fda217/Lib/shutil.py#L456

Reply:
yeah, I found out as much, but we can still give it a name in this script to help the reader :slightly_smiling_face:

Reply:
done

Reply:
actually, as far as I could test an `OSError` with `.winerror == 32` actually gets raised as a more specific `PermissionError`, but I guess the `.winerror == 32` specifies this is the "still in use" error. So I just suggest we add a `still_in_use_winerror = 32` and use that for readability